Formula/aerc.rb

```diff
@@ -0,0 +1,20 @@
+class Aerc < Formula
+  desc "Email client that runs in your terminal"
+  homepage "https://aerc-mail.org/"
+  url "https://git.sr.ht/~sircmpwn/aerc/archive/0.5.1.tar.gz"
+  sha256 "8feb7f87aa210ab7d8b271c34ad39af13f553447994341b3c25e7a659ce37d46"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "scdoc" => :build
+
+  def install
+    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
```

Comment:
This shouldn't be needed with modern go programs

Reply:
fixed this!